src/title-slide/slide-text.html

```diff
@@ -1,3 +1,4 @@
+<div style="display:block; float:right;"><img src="/images/l4gg-logo-500.png"></div>
```

Comment:
This image isn't getting picked up by webpack; I suggest using a "media url".  Take a look at the SlideMedia argument here: https://github.com/L4GG/timeline/blob/b3c48db09589c009f6f8b6d9c9d57f85c2efefd9/src/utils/eventDirectoryToSlide.js#L52

I can commit a patch directly to this branch if you'd like.